packages/contracts-bedrock/deploy-config/base-sepolia.json

```
@@ -0,0 +1,62 @@
+{
+  "finalSystemOwner": "0x608081689Fe46936fB2fBDF7552CbB1D80ad4822",
+  "portalGuardian": "0xA9FF930151130fd19DA1F03E5077AFB7C78F8503",
```

I think this is causing the failure:

```suggestion
  "portalGuardian": "0xA9FF930151130fd19DA1F03E5077AFB7C78F8503",
```

---

Committing this to see.

---

I am unclear what the diff is here

---

Ah ok github bug, the fix is already merged

---

deletes that line. The var name is now `superchainConfigGuardian`

---

and it's listed below.